apps/server/src/modules/tool/launch-tool/service/strategy/basic-tool-launch.strategy.ts

```diff
@@ -1,17 +1,21 @@
-import { ExternalToolConfigDO } from '@shared/domain';
 import { Injectable } from '@nestjs/common';
+import { EntityId } from '@shared/domain';
+import { LaunchRequestMethod, PropertyData, PropertyLocation } from '../../types';
 import { AbstractLaunchStrategy } from './abstract-launch.strategy';
-import { PropertyData, PropertyLocation } from '../../types';
-import { IToolLaunchStrategy } from './tool-launch-strategy.interface';
+import { IToolLaunchParams } from './tool-launch-params.interface';
 
 @Injectable()
-export class BasicToolLaunchStrategy extends AbstractLaunchStrategy implements IToolLaunchStrategy {
-	// eslint-disable-next-line @typescript-eslint/no-unused-vars
-	buildToolLaunchDataFromConcreteConfig(config: ExternalToolConfigDO): PropertyData[] {
-		return [];
+export class BasicToolLaunchStrategy extends AbstractLaunchStrategy {
+	public override buildToolLaunchDataFromConcreteConfig(
```

Comment:
A tok i see no better way to use this function in AbstractLaunchStrategy. 
But i do not like that it is fullfilled with something and return it. It should be enough that it return null and the point that use it must interpret that it should not be use.

Reply:
This change make to code not easier, but cleaner. Please keep it in mind if you refactor something. @MarvinOehlerkingCap